apps/ex_shop/test/models/line_item_test.exs

```diff
@@ -21,10 +21,28 @@ defmodule ExShop.LineItemTest do
       add_count: @max_master_quantity
     }
   }
-  @product_attr Map.merge(@product_data, @product_master_variant_data)
+  @variant_option_value_attrs %{
+    variant_option_values: [
+      %{
+        option_value_id: "1",
+        option_type_id: "1"
+      }
+    ]
+  }
+
+  @product_attr Map.merge(@product_data, @product_master_variant_data) |> Map.merge(@variant_option_value_attrs)
+
+  @valid_variant_attrs %{
+    cost_price: "120.5",
+    discontinue_on: %{"year" => "2016", "month" => "2", "day" => "1"},
+    height: "120.5", weight: "120.5", width: "120.5",
+    sku: "URG123"
+  }
 
-  @tag :pending
```

Comment:
please keep the pending test as we have to add more tests where we will re-check all the tests with non-master variants :)